bookshelf/bookshelf.d.ts

```diff
@@ -177,8 +191,12 @@ declare module 'bookshelf' {
 			keys() : string[];
 			last() : T;
 			lastIndexOf(value : any, fromIndex? : number) : number;
-			map(predicate? : Lodash.ListIterator<T, boolean>|Lodash.DictionaryIterator<T, boolean>|string, thisArg? : any) : T[];
+
+			// See https://github.com/Microsoft/TypeScript/blob/v1.8.10/lib/lib.core.es7.d.ts#L1122
+			map<U>(predicate? : Lodash.ListIterator<T, U>|string, thisArg? : any) : U[];
+			//map(predicate? : Lodash.ListIterator<T, boolean>|Lodash.DictionaryIterator<T, boolean>|string, thisArg? : any) : T[];
 			map<R extends {}>(predicate? : R) : T[];
```

Comment:
Are you talking about `map<R extends {}>(predicate? : R) : T[];`?
I don't know what this line does: I don't know how to write a test/example for it, that's why I didn't touch it.